game.py

```diff
@@ -130,26 +152,27 @@ def game_main():
                     message_combat = random_combat_messages[(rand.randint(0,4))]
                     print(message_combat, '\n')
 
-                    player.current_energy -= 1
+                    player.lose_energy(1)
+
                     for skill in skills_list:
                         if skill.name == "Combat":
                             skill.gain_xp(1)
-                            skill.level_check()
 
                     get_cur_stats(skills_list, player)
 
             else:
                 cprint("Not a Valid Input! Try again.", 'red')
                 
         elif player.current_energy == 0 and get_input != "rest":
-            cprint("You are too tired. Consider resting!", 'red') 
-            
+            cprint("You are too tired. Consider resting!", 'red')
+
         else:   
             message_rest = random_rest_messages[(rand.randint(0, 4))]
             print(message_rest)
 
             player.rest()
             get_cur_stats(skills_list, player)
 
+clear_term()    # Clear the terminal for new game        
```

Comment:
made more sense to have this run before the game starts instead of at the top of the file.